core/src/integrationTest/kotlin/ch/ergon/dope/clauses/DeleteIntegrationTest.kt

```diff
@@ -0,0 +1,62 @@
+package ch.ergon.dope.clauses
+
+import ch.ergon.dope.QueryBuilder
+import ch.ergon.dope.integrationTest.BaseIntegrationTest
+import ch.ergon.dope.integrationTest.TestCouchbaseDatabase.idField
+import ch.ergon.dope.integrationTest.TestCouchbaseDatabase.resetDatabase
+import ch.ergon.dope.integrationTest.TestCouchbaseDatabase.testBucket
+import ch.ergon.dope.resolvable.expression.unaliased.type.arithmetic.add
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isEqualTo
+import ch.ergon.dope.resolvable.fromable.useKeys
+import kotlin.test.AfterTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class DeleteIntegrationTest : BaseIntegrationTest() {
+    @AfterTest
+    fun reset() {
+        resetDatabase()
+    }
+
+    @Test
+    fun `delete single document and return id field`() {
+        val dopeQuery = QueryBuilder()
+            .deleteFrom(
+                testBucket.useKeys("employee:1"),
+            )
+            .returning(
+                idField,
+            ).build()
+
+        val actual = queryWithoutParameters(dopeQuery)
+        val actualQueryResult = actual.rows[0].contentAs<Map<String, Number>>()
+
+        assertEquals(mapOf("id" to 1), actualQueryResult)
+    }
+
+    @Test
+    fun `delete every document in bucket`() {
+        val deleteEverythingDopeQuery = QueryBuilder()
+            .deleteFrom(
+                testBucket,
+            )
+            .where(
+                1.add(1).isEqualTo(2),
```

Comment:
Even though you resolved this, I still wonder why you have this Equals expression and don't just use `true`?
or just remove the where?

Reply:
because in an integration we want to test as many things as possible
(please set the comment to 'unresolved' if you add a comment to a already resolved comment)